.gitignore

```diff
@@ -3,6 +3,7 @@
 target/
 *.log
 *.iml
+/out/
 
 bazel-*
```

Comment:
You could get rid of the bazel-* ignores, plus the BUILD and WORKSPACE files.

Reply:
Good catch, thanks!

Done